test/test_congress.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import responses
+from congress import Congress, Client, BaseParser, BaseValidator
+
+@pytest.fixture
+def congress():
+    api_key = 'testing_key'
+    return Congress(api_key)
+
+@pytest.fixture
+def client():
+    api_key = 'testing_key'
+    return Client(api_key, BaseParser(), BaseValidator())
+
+@responses.activate
+def test_all_members_getter(congress, client):
+    # Init testing values.
+    congress_no = '114'
+    chamber = 'senate'
+    n_members = 100
+    # Mock API response.
+    api_url = client.build_url(congress_no, chamber, "members.json")
+    mock_members = [dict()] * n_members
+    mock_results = [{'members': mock_members}]
```

Comment:
there are sample json responses on the propublica page, I would rather use an actual example.